install/disconnected_install.adoc

```diff
@@ -253,6 +253,11 @@ $ docker pull registry.access.redhat.com/rhgs3/rhgs-s3-server-rhel7:<tag>
 +
 [IMPORTANT]
 ====
+For Red Hat support, a CNS subscription is required for `rhgs3/` images.
```

Comment:
Are they supported only if you have the subscription, or do you need the subscription to obtain the images at all?

I'd spell out CNS.

Reply:
Thanks! They're only supported if you have the subscription.